.github/workflows/release.yml

```diff
@@ -43,3 +43,19 @@ jobs:
         run: lerna version --yes
         env:
           GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - name: Generate Lerna Versions PR
+        id: vpr
+        uses: peter-evans/create-pull-request@v4
```

Comment:
curious what this means- why keep peter-evans?

Reply:
@alex-sheehan-edx That's just the name of the Github Action; it represents the Github organization and repo name where the Github Action is defined: https://github.com/peter-evans/create-pull-request